packages/tx/CHANGELOG.md

```diff
@@ -6,6 +6,79 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 3.3.0 - 2021-07-02
+
+### Improved L2 Tx Support
+
+This tx release bumps the `Common` library dependency version to `v2.4.0` and is therefore assured to works with the reworked `Common.custom()` method which can be used for an easier instantiation of common custom chain instances for sending txs to a custom (L2) network.
```

Comment:
Very minor suggestion: `is therefore assured to work with`

Reply:
```suggestion
This tx release bumps the `Common` library dependency version to `v2.4.0` and is therefore assured to work with the reworked `Common.custom()` method which can be used for an easier instantiation of common custom chain instances for sending txs to a custom (L2) network.
```